File: Data Structures & Algorithms/max-consecutive-ones/submission-2.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int size=nums.size();
        int count=0;
        int maxcount=0;
        for(int i=0;i<size;i++){
            if(nums[i]==1){
                count++;
                if(maxcount<count){
                    maxcount=count;
                }
            }
            else{
                count=0;
            }

        }
        return maxcount;
    }
};

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-2.java
class Solution {
    public int[] productExceptSelf(int[] nums) {
        int []res= new int[nums.length];
        int prefix=1;
        int n=nums.length;
        for(int i=0;i<n;i++){
            res[i]=prefix;
            prefix*=nums[i];
        }
        int postfix=1;
        for(int i=n-1;i>=0;i--){
            res[i]=res[i]*postfix;
            postfix=postfix*nums[i];
        }
        return res;
    }
}  
